hibernate-core/src/test/java/org/hibernate/orm/test/dialect/H2DialectTestCase.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.orm.test.dialect;
+
+import java.util.List;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import org.hibernate.dialect.H2Dialect;
+import org.hibernate.orm.test.jpa.BaseEntityManagerFunctionalTestCase;
+
+import org.hibernate.testing.RequiresDialect;
+import org.hibernate.testing.TestForIssue;
+import org.junit.Test;
+
+import static org.hibernate.testing.transaction.TransactionUtil.doInJPA;
+import static org.junit.Assert.assertEquals;
+
+@RequiresDialect(H2Dialect.class)
+public class H2DialectTestCase extends BaseEntityManagerFunctionalTestCase {
```

Comment:
I don't understand why we need a new test.

Hibernate already has oodles of tests for `GenerationType.IDENTITY`. What is new/different about this one?

Reply:
Nothing special about this test class. 

I just did not see any test for H2 in `org.hibernate.orm.test.dialect` package and wrote one as requested in [CONTRIBUTING.md](https://github.com/hibernate/hibernate-orm/blob/main/CONTRIBUTING.md)

Should I remove this test ? 

Any other changes to be made ?

Reply:
Yes, please remove the test.

Reply:
removed

Reply:
Thanks.